### PR TITLE
Daily Update Portugues

### LINKED ISSUE #426: SSS Hentais - português
### Source name

SSS Hentais

### Source language

Português 

### Source link

There's no link

### Other details

We spoke to the owner of SSS scanlator and he confirmed that they no longer make nfsw, so sss Hentai has been shut down. 

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source website is down.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #428: AnimaRagia - português
### Source name

AnimaRegia

### Source language

Português 

### Source link

https://animaregia.net/

### Other details

The website ia down - forever and closed

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source website is down.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by domflamingo2**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [X] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [X] Have not changed source names
- [X] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension
- [X] Updated the extension url

Changelog:

HiperCool Change Url
CeriseScans Change Url
Arkham Scan -> HuntersScan (Fusion)
animaregia (Down) close 
SssScanlatorHentai (is closed)

Issues close #428
Issues close #426

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 1]
```diff
@@ -66,7 +66,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Cat300", "https://cat300.com", "th", isNsfw = true, className = "Cat300", overrideVersionCode = 1),
         SingleLang("CatOnHeadTranslations", "https://catonhead.com", "en", overrideVersionCode = 2),
         SingleLang("CBHentai", "https://cbhentai.net", "vi", isNsfw = true, overrideVersionCode = 7, pkgName = "hentaicube", className = "HentaiCB"),
-        SingleLang("Cerise Scan", "https://cerisescan.com", "pt-BR", pkgName = "cerisescans", isNsfw = true, overrideVersionCode = 7),
+        SingleLang("Cerise Scan", "https://cerisescan.net", "pt-BR", pkgName = "cerisescans", isNsfw = true, overrideVersionCode = 7),
```
Comment (vetleledaal) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt]: You need to bump the version when updating extensions.
```suggestion
        SingleLang("Cerise Scan", "https://cerisescan.net", "pt-BR", pkgName = "cerisescans", isNsfw = true, overrideVersionCode = 8),
```

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 2]
```diff
@@ -144,7 +144,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("HentaiZM", "https://manga.hentaizm.fun", "tr", isNsfw = true),
         SingleLang("HentaiZone", "https://hentaizone.xyz", "fr", isNsfw = true, overrideVersionCode = 1),
         SingleLang("HerenScan", "https://herenscan.com", "es"),
-        SingleLang("HipercooL", "https://hipercool.xyz", "pt-BR", isNsfw = true, className = "Hipercool"),
+        SingleLang("HipercooL", "https://hiper.cool", "pt-BR", isNsfw = true, className = "Hipercool"),
```
Comment (vetleledaal) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt]:
```suggestion
        SingleLang("HipercooL", "https://hiper.cool", "pt-BR", isNsfw = true, className = "Hipercool", overrideVersionCode = 1),
```

**Review by vetleledaal — CHANGES_REQUESTED** [review]
Also, combining multiple changes like this makes it harder to review.
Seems like all remaining icons were properly deleted.


Changes:
Cerise Scan (pt-BR): https://cerisescan.com -> https://cerisescan.net
HipercooL (pt-BR): https://hipercool.xyz -> https://hiper.cool
Arkham Scan (pt-BR): deleted, replaced by Hunters Scan (pt-BR) (existing extension)
SSS Hentais (pt-BR): deleted (Closes #426)
AnimaRegia (pt-BR): deleted (Closes #428)

**Comment by domflamingo2** [comment]
> Also, combining multiple changes like this makes it harder to review. Seems like all remaining icons were properly deleted.
> 
> Changes: Cerise Scan (pt-BR): https://cerisescan.com -> https://cerisescan.net HipercooL (pt-BR): https://hipercool.xyz -> https://hiper.cool Arkham Scan (pt-BR): deleted, replaced by Hunters Scan (pt-BR) (existing extension) SSS Hentais (pt-BR): deleted (Closes #426) AnimaRegia (pt-BR): deleted (Closes #428)

Thanks

**Review by beer-psi — APPROVED** [review]

**Comment by domflamingo2** [comment]
Please don't accept the pr, tomorrow I'll delete some extensions that don't work anymore.

**Comment by beer-psi** [comment]
> Please don't accept the pr, tomorrow I'll delete some extensions that don't work anymore.

Then make a new pull request for it. Vetle already recommended you to make **one PR for each extension.**

**Pull request merged**